CHANGELOG.md

```diff
@@ -1,3 +1,286 @@
+- Support accessibility text input [#1875](https://github.com/JetBrains/compose-multiplatform-core/pull/1875)
+- Support text input for UI Tests [#1875](https://github.com/JetBrains/compose-multiplatform-core/pull/1875)
+- Accessibility: added ability to traverse nodes within a scrollable container [#1837](https://github.com/JetBrains/compose-multiplatform-core/pull/1837)
+- Add ability to recreate Composable after `ComposeUIViewController` leaves view controller hierarchy [#1877](https://github.com/JetBrains/compose-multiplatform-core/pull/1877)
+- Support trackpad pointer input [#1893](https://github.com/JetBrains/compose-multiplatform-core/pull/1893)
+- Save Composable state when view controller leaves view controller hierarchy [#1904](https://github.com/JetBrains/compose-multiplatform-core/pull/1904)
+- Support text input when Full Keyboard Access is enabled [#1917](https://github.com/JetBrains/compose-multiplatform-core/pull/1917)
+- Added native behavior for tap, long tap and multiple tap to `BasicTextField(TextFieldState)` [#1923](https://github.com/JetBrains/compose-multiplatform-core/pull/1923)
+- Magnifier support for `BasicTextField(TextFieldState)` [#1926](https://github.com/JetBrains/compose-multiplatform-core/pull/1926)
+
+### Desktop
+
+- Improved performance on Windows by replacing the native code compiler to Clang (in 1.8.0 comparing to 1.7.3) [#1863](https://github.com/JetBrains/compose-multiplatform-core/pull/1863)
+  - Software rendering (used on some VMs) FPS is **6.397x higher** on average
+  - Reduction of installed application size. Example https://kmp.jetbrains.com:
+     - the installer size is changed from 44.9 Mb to 44.1 Mb
+     - the installed size is changed from 107 Mb to 103 Mb
+- The default ProGuard version is set to 7.7.0 [#5279](https://github.com/JetBrains/compose-multiplatform/pull/5279)
+   - If there are any new errors in the release build, update [the ProGuard rules](https://www.jetbrains.com/help/kotlin-multiplatform-dev/compose-native-distribution.html#minification-and-obfuscation)
+   - A usual workaround is to add `-keep class` for the associated with error class in "Location:"
+   - If the error contains `androidx.` package, it might a Compose bug, please report in https://youtrack.jetbrains.com/issues/CMP. The `-keep class` workaround should also work in this case
+
+### Web
+
+- Adds experimental PointerIcon.fromKeyword function to change the browser cursor [#1704](https://github.com/JetBrains/compose-multiplatform-core/pull/1704)
+
+### Resources
+
+- Add new API to preload and cache font and image resources on web targets: `preloadFont`, `preloadImageBitmap`, `preloadImageVector` [#5159](https://github.com/JetBrains/compose-multiplatform/pull/5159)
+
+### Gradle Plugin
+
+- Support compose resources in `androidLibrary` target [#5157](https://github.com/JetBrains/compose-multiplatform/pull/5157)
+
+### Navigation
```

Comment:
That's weird to aggregate notes between two unstable versions, but.. let's think how to do it properly next time 🙃